tests/test-frameworks/isolation/BUILD

```diff
@@ -3,106 +3,112 @@ load("@rules_scala_annex//rules:scala.bzl", "scala_library", "scala_test")
     isolation = "classloader",
-    scala = "//scala:2_12",
+    scala = "//scala:2_13",
     deps = [
-        "@annex_test//:org_specs2_specs2_common_2_12",
-        "@annex_test//:org_specs2_specs2_core_2_12",
-        "@annex_test//:org_specs2_specs2_matcher_2_12",
+        "@annex_test//:org_specs2_specs2_common_2_13",
+        "@annex_test//:org_specs2_specs2_core_2_13",
+        "@annex_test//:org_specs2_specs2_matcher_2_13",
     ],
 )
 
 scala_test(
     name = "classloader-shared",
     srcs = ["Test.scala"],
     isolation = "classloader",
-    scala = "//scala:2_12",
+    scala = "//scala:2_13",
     shared_deps = [":global"],
     deps = [
         ":global",
-        "@annex_test//:org_specs2_specs2_common_2_12",
-        "@annex_test//:org_specs2_specs2_core_2_12",
-        "@annex_test//:org_specs2_specs2_matcher_2_12",
+        "@annex_test//:org_specs2_specs2_common_2_13",
+        "@annex_test//:org_specs2_specs2_core_2_13",
+        "@annex_test//:org_specs2_specs2_matcher_2_13",
     ],
 )
 
 scala_test(
     name = "none",
     srcs = ["Test.scala"],
-    scala = "//scala:2_12",
+    scala = "//scala:2_13",
     deps = [
         ":global",
-        "@annex_test//:org_specs2_specs2_common_2_12",
-        "@annex_test//:org_specs2_specs2_core_2_12",
-        "@annex_test//:org_specs2_specs2_matcher_2_12",
+        "@annex_test//:org_specs2_specs2_common_2_13",
+        "@annex_test//:org_specs2_specs2_core_2_13",
+        "@annex_test//:org_specs2_specs2_matcher_2_13",
     ],
 )
 
 scala_test(
     name = "none-jvm",
     srcs = ["TestJvm.scala"],
     isolation = "none",
-    scala = "//scala:2_12",
+    scala = "//scala:2_13",
     deps = [
-        "@annex_test//:org_specs2_specs2_common_2_12",
-        "@annex_test//:org_specs2_specs2_core_2_12",
-        "@annex_test//:org_specs2_specs2_matcher_2_12",
+        "@annex_test//:org_specs2_specs2_common_2_13",
+        "@annex_test//:org_specs2_specs2_core_2_13",
+        "@annex_test//:org_specs2_specs2_matcher_2_13",
     ],
 )
 
 scala_test(
     name = "process",
     srcs = ["Test.scala"],
     isolation = "process",
-    scala = "//scala:2_12",
+    scala = "//scala:2_13",
     deps = [
         ":global",
-        "@annex_test//:org_specs2_specs2_common_2_12",
-        "@annex_test//:org_specs2_specs2_core_2_12",
-        "@annex_test//:org_specs2_specs2_matcher_2_12",
+        "@annex_test//:org_specs2_specs2_common_2_13",
+        "@annex_test//:org_specs2_specs2_core_2_13",
+        "@annex_test//:org_specs2_specs2_matcher_2_13",
     ],
 )
 
 scala_test(
-    name = "process-2_11",
+    name = "process-2_12",
     srcs = ["Test.scala"],
     isolation = "process",
-    scala = "//scala:2_11",
+    scala = "//scala:2_12",
     deps = [
-        ":global",
-        "@annex_test//:org_specs2_specs2_common_2_11",
-        "@annex_test//:org_specs2_specs2_core_2_11",
-        "@annex_test//:org_specs2_specs2_matcher_2_11",
+        ":global-2-12",
```

Comment:
Agreed. This is a bug that we didn't catch because 2.12 and 2.11 aren't as different as 2.12 and 2.13. Changing this to the 2.13 global causes compilation to fail. So, I'm going to consider this a bug fix and leave the change.